excel/src/main/java/tech/tablesaw/io/xlsx/XlsxReader.java

```diff
@@ -147,7 +147,7 @@ private Boolean isBlank(Cell cell) {
       case NUMERIC:
         if (DateUtil.isCellDateFormatted(cell)
             ? cell.getDateCellValue() != null
-            : cell.getNumericCellValue() != 0) {
+            : cell.getNumericCellValue() != 0 || cell.getCellType() == NUMERIC) {
```

Comment:
`cell.getCellType() == NUMERIC` will always be `true` here, so this is the same as rewriting the entire expression to:
```
if (!DateUtil.isCellDateFormatted(cell) || cell.getDateCellValue() != null)
```